src/components/TripPlanner/TripPlanner.scss

```diff
@@ -0,0 +1,53 @@
+// @import"../App.scss";
+
+// Temporary border constants for designing. Remove or set to transparent 0px after use
+$border: 2px solid rgb(255, 0, 0);
+
+.tripplanner-container {
+    height: 90vh;
+    width: 1440px;
+    display: grid;
+    grid-template-rows: 10% 10% 40% 40%;
+    justify-items: center;
+    border: $border;
+}
+
+.tripplanner-header {
+    border: $border;
+    margin: 0.5%;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    display: grid;
+    align-items: center;
+    font-size: 2em;
+}
+
+.tripplanner-search {
+    border: $border;
+    width: 10%;
+    margin: 0.5%;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    display: grid;
+    align-items: center;
+}
+
+.tripplanner-map {
+    border: $border;
+    width: 50%;
+    margin: 2%;
+    grid-row-start: 3;
+    grid-row-end: 4;
+    display: grid;
+    align-items: center;
+}
+
+.tripplanner-table {
+    border: $border;
+    margin: 2%;
+    width: 80%;
+    grid-row-start: 4;
+    grid-row-end: 5;
+    display: grid;
+    align-items: center;
+}
```

Comment:
Standard practice is to have exactly one empty line at the end of files, if you don't have one then GitHub gives you this cheerful ⛔ sign to let you know!